src/main/java/com/xebialabs/overcast/host/CachedLibvirtHost.java

```diff
@@ -86,7 +86,7 @@
         List<Filesystem> filesystemMappings, List<String> copySpec) {
         super(libvirt, baseDomainName, ipLookupStrategy, networkName, startTimeout, bootDelay, filesystemMappings);
         this.provisionUrl = checkNotNullOrEmpty(provisionUrl, "provisionUrl");
-        this.provisionCmd = checkNotNullOrEmpty(provisionCmd, "provisionCmd");
+        this.provisionCmd = checkNotNullOrEmpty(provisionCmd, "provisionCmd").trim();
```

Comment:
Ric's point is that a `.trim()` might actually make you end up with an empty string.